lib/rubocop/cop/layout/empty_line_after_guard_clause.rb

```diff
@@ -85,8 +86,10 @@ def contains_guard_clause?(node)
           node.if_branch&.guard_clause?
         end
 
-        def next_line_empty?(line)
-          processed_source[line].blank?
+        def next_line_empty_or_rubocop_enable_comment?(line)
```

Comment:
I think we have to still hold empty line after directive comment. What do you think?

Reply:
Yeah, I agree. There should still be an empty line, but it should come right after the enable directive.